src/e2e/kotlin/fi/aalto/cs/apluscourses/e2e/PlaceholderTest.kt

```diff
@@ -55,5 +57,25 @@ class PlaceholderTest {
         }
       }
     }
+    //step 8
+    step("Authenticate") {
+      with(ideFrame()) {
+        menu().select("A+")
+        menu().select("Set A+ Token")
+      }
+      with(dialog("A+ Token")) {
+        jPasswordField().text = System.getenv("APLUS_TEST_TOKEN")
+        button("OK").click()
+      }
+      with(ideFrame()) {
+        with(assignments()) {
+          waitFor(
+            Duration.ofSeconds(60),
+            Duration.ofSeconds(1),
```

Comment:
why not `Duration.ofSeconds(61)`?

Reply:
the first duration is the timeout and the second one is how often it's checked